File: 2022/Leetcode/problem412.java
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> answer = new ArrayList<String>();

        for (int i = 1; i <= n; i++) {
            if (i % 15 == 0) {
                answer.add("FizzBuzz");
            } else if (i % 3 == 0) {
                answer.add("Fizz");
            } else if (i % 5 == 0) {
                answer.add("Buzz");
            } else {
                answer.add(String.valueOf(i));
            }
        }

        return answer;
    }
}

File: 2022/Leetcode/problem412.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        bool isNum;
        string result;
        vector<string> answer(n);
        
        for (int i = 1; i <= n; i++) {
            result = "";
            isNum = true;
            
            if (i % 3 == 0) {
                result += "Fizz";
                isNum = false;
            }
            
            if (i % 5 == 0) {
                result += "Buzz";
                isNum = false;
            }
            
            if (isNum) {
                result = to_string(i);
            }
            
            answer[i - 1] = result;
        }
        
        return answer;
    }
};